.github/workflows/ensure-changelog-entry.yml

```diff
@@ -0,0 +1,129 @@
+name: Check change log entry
+on:
+  pull_request:
+    types: [opened, reopened, edited]
+
+jobs:
+  check:
+    permissions:
+      pull-requests: write
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Check membership
+        id: membership
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            // Is author of the PR a member of the DataDog Org.
+            // NOTE: https://docs.github.com/en/rest/orgs/members?apiVersion=2022-11-28#check-organization-membership-for-a-user
+            try {
+              response = await github.rest.orgs.checkMembershipForUser({
+                org: context.repo.owner,
+                username: context.payload.sender.login
+              })
+
+              isMember = response.status == 204
+            } catch (e) {
+              core.warning("An error occured on membership check, see console output");
+              console.log(e)
+
+              isMember = false
+            }
+
+            return isMember
+
+      - name: Find existing comment
+        id: comment
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const options = github.rest.issues.listComments.endpoint.merge({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: context.payload.pull_request.number,
+            });
+            const comments = await github.paginate(options)
+            const comment = comments.find((cmnt) => {
+              return cmnt.body.startsWith("<!-- HIDDEN_MARKER -->")
```

Comment:
Maybe add "change log comment" to the "hidden marker" in case we need more than one marker?